airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml

```diff
@@ -14,14 +14,15 @@ tests:
   basic_read:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/configured_catalog.json"
-      timeout_seconds: 600
+      timeout_seconds: 3600
```

Comment:
@sherifnada you can see it here for example

Reply:
Set this to 1200, and INSIGHTS_DAYS_PER_JOB to 7, instead of 1.